method/application/get.js

```diff
@@ -0,0 +1,63 @@
+const pull = require('pull-stream')
+module.exports = function GroupApplicationGet (server) {
+  return function groupApplicationGet (applicationId, cb) {
+    const query = [
+      {
+        $filter: {
+          dest: applicationId
+        }
+      }
+    ]
+
+    server.get({ id: applicationId, private: true }, (rootErr, rootData) => {
+      if (rootErr) cb(rootErr)
+      pull(
+        server.backlinks.read({ query }),
+        pull.collect((err, data) => {
+          if (err) cb(err)
+          let final = {
+            root: applicationId,
+            applicantId: rootData.author,
+            groupId: rootData.content.groupId,
+            recps: rootData.content.recps
+          }
+          /* If there's only the root message */
+          if (!data || data.length < 1) {
+            const res = {
+              ...final,
+              ...rootData.content,
+              text: [rootData.content.text.append || '']
```

Comment:
I would be paranoid and put `(rootData.content.text && rootData.content.text.append) || ''

also why is the text and Array?

My preference would be to see the data structure this `get` is returning in the README. This currently forces a person to try the method to see, or to read across this code and understand how this reduce is working

Reply:
Oh I see why it's an Array. hmmm,
I think this will work for now yes

Reply:
Added to README :+1: